ttnn/cpp/ttnn/operations/eltwise/binary/device/kernels/dataflow/reader_bcast_h_interleaved.cpp

```diff
@@ -0,0 +1,76 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include "dataflow_api.h"
+
+void kernel_main() {
```

Comment:
1. Are we just copying files over from a different directory?
2. What is the plan for the two sets of files?
3. Are they supposed to diverge or are they supposed to be the same?
4. If they are the supposed to be the same then why not delete the old ones?
5. If they are supposed to be the same and there's a good reason for having two, you need clear distinctive comments to make sure the two stay in sync.

Reply:
1. For now we are just copying them over because I wanted to be able to have the CircularBufferConfig to go into location 2 rather than 16 to get what we believe to be a boost in performance.  The idea was that during dispatching we do not want the overhead of copying over unused circular buffer configurations where possible.  
2. My expectation is that this code would be deleted once Yan's broadcasting feature goes in.
3. Yeah they diverge because the output index is no longer 16 for ow.
4. We have a lot of existing tests with current code.  The intent of this change was to allow the api to support this for now.  We are prioritizing making the API better and keeping the scope to that objective rather than cleaning up al the internal code as the API is being changed.
5. I'll add a comment here.